scoringrules/core/interval/_score.py

```diff
@@ -40,6 +40,8 @@ def _weighted_interval_score(
     B = backends.active if backend is None else backends[backend]
     K = weight_alpha.shape[0]
     IS = _interval_score(obs, lower, upper, alpha)
-    WIS = B.sum(IS * [None, weight_alpha], axis=1) + weight_median * median
+    WIS = (
+        B.sum(IS * B.expand_dims(weight_alpha, axis=0), axis=1) + weight_median * median
```

Comment:
`expand_dims(weight_alpha, axis=-2)` would be more robust here imo

Reply:
Then we probably also want to sum along -1 right?